features/support/rummager_helpers.rb

```diff
@@ -0,0 +1,25 @@
+require "gds_api/test_helpers/rummager"
+
+module RummagerHelpers
+  include GdsApi::TestHelpers::Rummager
+
+  def stub_rummager
+    stub_any_rummager_post
+  end
+
+  def assert_policy_published_to_rummager(policy)
+    expected_json = JSON.parse({
```

Comment:
What's the reason for the JSON.parse(foo.to_json)?

This might be an appropriate place for a code comment to explain what this is for, if it's not just an oversight.